File: BubbleSort.java
public class BubbleSort {
    public int[] sort(int mass[]){
        int count = 0;
        while (count < mass.length) {
            int swap;
            for (int i = 0; i < mass.length - 1; i++) {
                if (mass[i] > mass[i + 1]) {
                    swap = mass[i];
                    mass[i] = mass[i + 1];
                    mass[i + 1] = swap;
                }
            }
            count++;
        }
        return mass;
    }
}

File: BubbleSort.cs
namespace Chapter1
{
    public class BubbleSort
    {
        public static int[] Sort(int[] mass)
        {
            int middle;
            for (int i = 0; i < mass.Length; i++)
            {
                for (int x = 0; x < mass.Length - 1; x++)
                {
                    if (mass[x] > mass[x + 1])
                    {
                        middle = mass[x];
                        mass[x] = mass[x + 1];
                        mass[x + 1] = middle;
                    }
                }
            }
            return mass;
        }
    }
}